_config.yml

```diff
@@ -15,6 +15,7 @@ download_file_names:
   deb-headless:              "jamulus_headless_3.7.0_ubuntu_amd64.deb"
   windows:                   "jamulus_3.7.0_win.exe"
   mac:                       "jamulus_3.7.0_mac.dmg"
+  mac-lagacy:                "jamulus_3_7_0_mac_legacy.dmg"
```

Comment:
Should be:
`mac-legacy`

Reply:
> Should be:
> `mac-legacy`

Hrm, why?

https://github.com/jamulussoftware/jamulus/pull/1768/files#diff-55d631a22c9e350832273ee0abf8a6013435fa6051789cbe311b981b2ec0f348R132
https://github.com/jamulussoftware/jamulus/pull/1768/files#diff-ff6c0967f1173b1c4de6ea1230b43caaebe9c69607ca8e6dbb4b37f5ee3c6ec3R6